test/rekt/resources/kafkachannel/kafkachannel.go

```diff
@@ -84,3 +97,43 @@ func WithRetentionDuration(retentionDuration string) manifest.CfgFn {
 		cfg["retentionDuration"] = retentionDuration
 	}
 }
+
+// AsRef returns a KRef for a Channel without namespace.
+func AsRef(name string) *duckv1.KReference {
+	return &duckv1.KReference{
+		Kind:       EnvCfg.ChannelGK,
+		APIVersion: EnvCfg.ChannelV,
+		Name:       name,
+	}
+}
+
+// AsRef returns a KRef for a Channel without namespace.
+func AsDestinationRef(name string) *duckv1.Destination {
+	return &duckv1.Destination{
+		Ref: &duckv1.KReference{
+			Kind:       EnvCfg.ChannelGK,
+			APIVersion: EnvCfg.ChannelV,
+			Name:       name,
+		},
+	}
+}
+
+// Address returns a Channel's address.
+func Address(ctx context.Context, name string, timings ...time.Duration) (*duckv1.Addressable, error) {
+	return addressable.Address(ctx, GVR(), name, timings...)
+}
+
+// ValidateAddress validates the address retured by Address
+func ValidateAddress(name string, validate addressable.ValidateAddress, timings ...time.Duration) feature.StepFn {
```

Comment:
I think you might need to look at the change made by @creydr here: https://github.com/knative-extensions/eventing-kafka-broker/pull/3412/commits/c5a0002bce5b4d120379b8b3715281a81da25d15 for this to work